src/bucket/BucketManager.h

```diff
@@ -340,19 +340,18 @@ class BucketManager : NonMovableOrCopyable
     // Scans BucketList for non-live entries to evict starting at the entry
     // pointed to by EvictionIterator. Evicts until `maxEntriesToEvict` entries
     // have been evicted or maxEvictionScanSize bytes have been scanned.
-    void startBackgroundEvictionScan(uint32_t ledgerSeq, uint32_t ledgerVers,
-                                     SorobanNetworkConfig const& cfg);
+    void startBackgroundEvictionScan(SearchableSnapshotConstPtr appliedState,
```

Comment:
nit: Could you please add a comment on what 'appliedState' is? I don't think we use this name anywhere, and it has pretty tricky semantics. Honestly, this could just be `lclSnapshot`, it doesn't like we use anything else but lcl for this.